docs/_pages/releases.md

```diff
@@ -13,6 +13,7 @@ sidebar:
 
 **Fixes**
 * Sometimes `BeEquivalentTo` reported an incorrect message when a dictionary was missing a key - [1454](https://github.com/fluentassertions/fluentassertions/pull/1454)
+* Some dictionary failures did not honor the user-provided reason - [1456](https://github.com/fluentassertions/fluentassertions/pull/1456)
```

Comment:
```suggestion
* Some dictionary failures did not honor the user-provided reason - [#1456](https://github.com/fluentassertions/fluentassertions/pull/1456)
```

Reply:
I know. I couldn't find a test covering that test.

Reply:
Fine with me, just wanted to check.